cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -177,6 +196,10 @@ class DownloaderCommand : CliktCommand(name = "download", help = "Fetch source c
     override fun run() {
         val failureMessages = mutableListOf<String>()
 
+        val licenseCategorizations =
+            licenseClassificationsFile.takeIf { it.isFile }?.readValue<LicenseClassifications>().orEmpty()
+                .categorizations
```

Comment:
Sometimes IDEA's auto-formatting is odd. Why not just a single line-break like

        val licenseCategorizations = licenseClassificationsFile.takeIf { it.isFile }
            ?.readValue<LicenseClassifications>().orEmpty().categorizations